app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class='col-6 mx-auto text-white'>
+<h2>Resend confirmation instructions</h2>
+
+<%= bootstrap_form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  <%= f.submit "Resend confirmation instructions" %>
+<% end %>
```

Comment:
- [ OPTINAL ] I think it would be great if you stick with the neak peak as much as possible so please acoording to the requirements fix the design to be the same since the log in button have width like the inputs so kindly fix it 🚀🔥💯 and I am assiging it as optional because the previous reviewer didn't mention it but still important t do it 🤩💪🏻

- Your
![image](https://github.com/anderCM/blog-app/assets/58553711/3770e2f2-77bd-4f78-aab0-e50f5854ecec)

- Given
![image](https://github.com/anderCM/blog-app/assets/58553711/34159034-6fa2-42eb-838a-c330a17c0a85)